content/2019-03-02_pyladies-snake-report.md

```diff
@@ -0,0 +1,180 @@
+  of time.
+* Filter out people who'd drop out after the first few lessons of the course.
+* Give an overview of Python's the basic data types and building blocks,
+  which should be helpful because the 3-month course is “bottom-up” (introduces
+  elementary concepts and builds on them), so the “big picture” only reveals
+  itself near the end.
+
+[Snake]: https://en.wikipedia.org/wiki/Snake_(video_game_genre)
+
+![See the coaches coaching!]({filename}/images/snake2019-20190202_152819.jpg)
+
+
+## The Saturday
+
+The last instance of the Snake workshop happened on Saturday, February 2nd.
+This was the 4th run so far (after the first trial for PyLadies in
+Hradec Králové in March 2018, one for PyLadies Brno in September
+2018, and one open for anyone in October 2018).
+
+For this workshop, Red Hat provided the venue (Plutonium & Neptunium meeting
+rooms in the TPB-C office) and food (of which there was a lot; thank you!).
+
+I was the lecturer and I handled the Red Hat sponsorship.
+
+We had an almost full house – more than 39 attendees plus 8
+coaches/organizers.
+(The listed capacity of the rooms is 40, but we fit comfortably.)
+
+We spent the morning (well, until 14:00) explaining basic concepts (numbers,
+expressions, strings, functions, conditions, lists, dicts, imports, loops, …)
+and the afternoon with the game.
+Based on the feedback to date, everyone enjoyed the show.
+There were smiles throughout; the coaches were kind and helpful, and I'd say
+the attendees learned as much as they could in a day.
+
+![The importance of good ol' pen & paper]({filename}/images/snake2019-20190202_162002.jpg)
+
+Of course, nothing is perfect – and there's a lot to improve for next time.
+
+
+### What didn't work
+
+I didn't manage time well, and I'd need about 30 more minutes at the end.
+Some people were already leaving, so I rushed through the end – which didn't
+help either the attendees or the coaches.
+(For next time, I already have a plan to shuffle the materials around and
+drop 30 minutes of less interesting stuff.)
+
+Also, there was too much food in the end.
+We had a generous sponsorship from Red Hat *and* people ordered their own
+lunches.
+This one is entirely on me as the link between the sponsor and the organizers;
+next time I will be coordinate better.
+
+Finally, I did not request spare laptops in time.
+Usually, Red Hat Regional IT is happy to lend a few out-of-warranty machines,
+which are very helpful when some attendee's computer stops working, turns
+out to be too slow, or has glitchy graphics.
+I forgot to request them in a reasonable time, so we didn't have them.
+Thankfully, they weren't needed – but hardware problems could have ruined the
+atmosphere of the workshop.
+
+![Not quite there yet]({filename}/images/snake2019-20190202_173055.jpg)
+
+
+### And as for me...
+
+I think I'm addicted to this workshop.
+
+It ticks the tree factors of intrinsic motivation are autonomy (deciding what
```

Comment:
"It ticks the tree factors of intrinsic motivation are autonomy ..." reads strange